.github/workflows/main.yml

```diff
@@ -62,7 +108,7 @@ jobs:
         run: ls -alFR
       - name: Create release and upload artifacts
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.DEPLOYTOKEN }}
```

Comment:
Will need to be reset to Your Repo deploy token.

Reply:
What in the world is a deploy token?

Reply:
Just the name of the token i use to deploy.

Reply:
Just `GITHUB_TOKEN` should have worked in your repository as well.